config/settings.yml

```diff
@@ -19,3 +19,8 @@ plugins:
     description:
       en: "Activate to set repeating spaces (U+0020) to non-breaking spaces (U+00A0).
       Note that discourse_normalize_whitespace must be disabled to preserve non-breaking spaces."
+  enable_bbscript:
+    default: true
```

Comment:
[dust] I would say that it might be good if bbscript is off by default given this may potentially be consumed by other clients than just RPNs. Then those site admins have the opportunity to understand the feature and decide whether they want it or not.

Reply:
ahh, thats true. I imagined other clients would fork and heavily modify our repo, but definitely should reduce the risk of them unknowingly enabling certain functions